### ISSUE TITLE
Share the python Engine

### ISSUE THREAD

**Issue by NoelJacob** [extism/python-pdk#33]
Is there a way for compiling the freezed and initialised python engine wasm and run arbritary script at runtime? Or run a somehow compiled script at runtime? I am basically looking for a python intrepreter sandboxing option.

**Comment by zshipko** [comment]
That is a neat idea! For now, I think you could just run `eval`/`exec` from inside a plugin. We've also been discussing ways of supporting something like this using a dynamically linked runtime, but haven't really started on that yet.

**Comment by nilslice** [comment]
@zshipko, this would mean the `input` passed into the `plugin.call` from the host would actually be the Python code to `eval`/`exec` correct?

**Comment by zshipko** [comment]
Yeah thanks for clarifying, the Python code would be the input to a plugin function that passes it to eval.

**Comment by gregfurman** [comment]
FYI, we ended up using the WASM binary released by [vmware-labs/webassembly-language-runtimes](https://github.com/vmware-labs/webassembly-language-runtimes).

Specifically, we're using the `python-3.12.0.wasm` artifact at https://github.com/vmware-labs/webassembly-language-runtimes/releases/latest/download/python-3.12.0.wasm

Hope this helps!
